server/src/server.rs

```diff
@@ -20,19 +20,21 @@ use crate::{distributed_engine::DistributedEngine, storage_engine::StorageEngine
 macro_rules! EngineErr2Status {
     ($e:expr) => {
         match $e {
-            EngineError::IO => -libc::EIO,
-            EngineError::NoEntry => -libc::ENOENT,
-            EngineError::NotDir => -libc::ENOTDIR,
-            EngineError::IsDir => -libc::EISDIR,
-            EngineError::Exist => -libc::EEXIST,
-            EngineError::NotEmpty => -libc::ENOTEMPTY,
+            EngineError::IO => libc::EIO,
+            EngineError::NoEntry => libc::ENOENT,
+            EngineError::NotDir => libc::ENOTDIR,
+            EngineError::IsDir => libc::EISDIR,
+            EngineError::Exist => libc::EEXIST,
+            EngineError::NotEmpty => libc::ENOTEMPTY,
+            // temp
+            EngineError::Path => -1,
+            EngineError::StdIo(_) => -1,
+            EngineError::Rocksdb(_) => -1,
```

Comment:
Why the error codes are all -1?

Reply:
Maybe we can use numbers >1000?

Reply:
I think we don't specify these values now.